repos/spack_repo/builtin/packages/ninja/package.py

```diff
@@ -26,6 +26,7 @@ class Ninja(Package):
 
     version("kitware", branch="features-for-fortran", git="https://github.com/Kitware/ninja.git")
```

Comment:
I think it is a good idea to mark this as deprecated since the Ninja upstream supports `pipe` and `fifo`, where as this only supports `pipe`.